src/test/java/com/synopsys/integration/stepworkflow/StepWorkflowTest.java

```diff
@@ -53,23 +52,25 @@ public void setup() {
     }
 
     @Test
-    public void testSuccessfulSingleStepDataStepWorkflow() {
+    public void testSuccessfulSingleStepDataStepWorkflow() throws Exception {
```

Comment:
If a test isn't supposed to throw an exception, we'd rather catch it and Assertions.fail with a meaningful message than throw it to fail the test-- if we just throw it, it means that whomever sees the test failure will think something in the tested code is broken, rather than the test code.